api/services/cartService.js

```diff
@@ -0,0 +1,9 @@
+const { cartDao } = require("../models");
+
+const createCart = async (userId, productId, quantity, price) => {
+  await cartDao.createCart(userId, productId, quantity, price);
+};
```

Comment:
```suggestion
const createCart = async (userId, productId, quantity, price) => {
  return cartDao.createCart(userId, productId, quantity, price);
};
```
1. insert, update, delete 와 같은 dml을 사용할 때에는 mysql driver에서 okpacket (insertId, affectedRow 등등이 담겨있는 객체) 이라는 것을 반환해줍니다. 우리 코드에서는 이를 활용하고 있지 않지만 활용하는 경우도 있기 떄문에 함수 내부에서 return으로 결과값을 반환해주도록 수정되면 좋을 것 같습니다.
2. 추가적으로 실행할 코드가 없으므로 await가 필요하지 않은 코드입니다.